src/Build/BackEnd/Components/SdkResolution/NuGet/NuGetSdkResolverBase.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Shared;
+using System;
+using System.IO;
+using System.Reflection;
+#if !FEATURE_APPDOMAIN
+using System.Runtime.Loader;
+#endif
+
+using SdkResolverBase = Microsoft.Build.Framework.SdkResolver;
+
+namespace Microsoft.Build.BackEnd.SdkResolution.NuGet
+{
+    /// <summary>
+    /// Acts as a base class for the NuGet-based SDK resolver and handles assembly resolution to dynamically locate NuGet assemblies.
+    /// </summary>
+    internal abstract class NuGetSdkResolverBase : SdkResolverBase
+    {
+        /// <summary>
+        /// The sub-folder under the Visual Studio installation where the NuGet assemblies are located.
+        /// </summary>
+        public const string PathToNuGetUnderVisualStudioRoot = @"Common7\IDE\CommonExtensions\Microsoft\NuGet";
```

Comment:
Is this documented? How will we know if it changes?

Reply:
I have no idea, maybe @emgarten knows?

Reply:
This is considered a well known location and it is defined in the VSIX manifest, we have it set here also: https://github.com/NuGet/NuGet.BuildTasks/blob/1d8af3499f94a32c1d128a42faceade39c1f4337/src/Microsoft.NuGet.Build.Tasks/ImportBeforeAfter/Microsoft.NuGet.ImportAfter.targets#L14

I would not expect this to change, if it did it would be very hard to find where it moved to.